package.json

```diff
@@ -13,13 +13,28 @@
   },
   "homepage": "https://github.com/alphagov/govuk-design-system#readme",
   "scripts": {
+    "build": "node tasks/build.js",
+    "start": "node tasks/serve.js",
     "lint": "gulp scss:lint"
   },
   "dependencies": {
     "@govuk-frontend/all": "^0.0.18-alpha"
   },
   "devDependencies": {
     "gulp": "^3.9.1",
-    "gulp-sass-lint": "^1.3.4"
+    "gulp-sass-lint": "^1.3.4",
+    "jstransformer-markdown": "^1.2.1",
```

Comment:
are these two jstransformer needed?

Reply:
yes, we need them to be used by `metalsmith-in-place` for rendering markdown and nunjucks syntax. the reason why they are not dependencies of `metalsmith-in-place` is because you can add whatever you transformer you want, rather then having all of them imported by `metalsmith-in-place`.